pallets/elections/src/lib.rs

```diff
@@ -59,6 +59,9 @@ pub mod pallet {
     #[pallet::storage]
     pub type MembersPerSession<T> = StorageValue<_, u32, ValueQuery>;
 
+    #[pallet::storage]
+    pub type ReservedMembers<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
```

Comment:
Added a small doc with terminology and storages. `Members` are confusing because actually are used in multiple contexts. With the upcoming cleanup you proposed this will go away (for now I left the todo mark)